python/lsst/ts/observatory/control/auxtel/atcalsys.py

```diff
@@ -317,13 +317,19 @@ async def _calculate_electrometer_exposure_times(
                     * electrometer_time_separation_vs_integration
                 ) + electrometer_integration_overhead
                 max_exp_time = electrometer_buffer_size * time_sep
-                if exptime > max_exp_time:
-                    electrometer_exptimes.append(max_exp_time)
+                if exptime < 1.0:
```

Comment:
can you make this "minimum exposure time" an attribute like the others at the top of the method, e.g. `eletrometer_mini_exp_time = 1.0`?